examples/mujoco/runnable/ant_v2_ddpg.py

```diff
@@ -72,8 +72,7 @@ def test_ddpg(args=get_args()):
         actor, actor_optim, critic, critic_optim,
         action_range=[env.action_space.low[0], env.action_space.high[0]],
         tau=args.tau, gamma=args.gamma,
-        exploration_noise=GaussianNoise(sigma=args.exploration_noise),
-        reward_normalization=True)
+        exploration_noise=GaussianNoise(sigma=args.exploration_noise))
     # collector
     train_collector = Collector(
         policy, train_envs,
```

Comment:
How about removing runnable/ files in this PR?

Reply:
Up to you